doc/nrf/releases_and_maturity/migration/migration_guide_2.6.rst

```diff
@@ -676,6 +676,86 @@ The following changes are recommended for your application to work optimally aft
   This change enables the use of standard |NCS| tools and procedures for building and configuring the controller.
   The SoftDevice Controller will for some time not have a QDID for the ISO operation.
 
+* For the Bluetooth Mesh samples and applications, a new sensor API (see :ref:`bt_mesh_sensors_readme`) is introduced with |NCS| v2.6.0.
+  The previous sensor API is deprecated.
+
+  The usage of the new sensor API is demonstrated in samples :ref:`bluetooth_mesh_sensor_client`, :ref:`bluetooth_mesh_sensor_server` and :ref:`bluetooth_mesh_light_lc`.
+
+  The Kconfig option :kconfig:option:`CONFIG_BT_MESH_SENSOR_USE_LEGACY_SENSOR_VALUE` is enabled by default in the deprecation period.
+  This means that the existing samples and applications can continue using the deprecated sensor API as normal during the deprecation period, without the additional configuration.
+  The samples and applications will get a deprecation warning when compiled, that the user can choose to disregard.
+
+  When the deprecation period is over, the deprecated sensor API will be removed, and the samples and applications will no longer compile unless updated to the new sensor API.
+
+  New and existing applications that want to use the new Sensor APIs must set :kconfig:option:`CONFIG_BT_MESH_SENSOR_USE_LEGACY_SENSOR_VALUE` to `=n` in their configuration.
+  This configuration value will be removed when the deprecation periode is over, and the configuration value then need to be removed from the application configuration.
+  It is important to check if :kconfig:option:`CONFIG_FPU` is set to `=y` to support accelerated floating point operations, and possible add :kconfig:option:`CONFIG_CBPRINTF_FP_SUPPORT` to support floating point printing.
+
+  * Sensor API arguments and callback parameters previously defined with :c:struct:`sensor_value` now uses :c:struct:`bt_mesh_sensor_value` instead.
+  * The :c:struct:`bt_mesh_sensor_value`.format needs to be filled by the application for variables passed to the Sensor API.
+  * There are several different types of :c:struct:`bt_mesh_sensor_format` described in the :file:`include/bluetooth/mesh/sensor_types.h`.
+  * When filling in sensor values for a channel the format can be found through ``sensor->type.channels[i].format`` defined for the given :c:struct:`bt_mesh_sensor` ``sensor``.
+  * The :c:struct:`bt_mesh_sensor_format` with a valid `format` can be converted to and from integer, float and :c:struct:`sensor_value` through `bt_mesh_sensor_value_to/from*` functions.
+  * Where the applications previously just added values directly to :c:struct:`sensor_value`.val1 and .val2, now the correct way is to use `bt_mesh_sensor_value_to/from*` functions to either set/extract values.
```

Comment:
Should :c:struct:`bt_mesh_sensor_format` references also be :c:member:`bt_mesh_sensor_value.format`? Or is this something else?

Reply:
`bt_mesh_sensor_value.format` is `* struct bt_mesh_sensor_format`, so one is the struct itself, the other is the field on the `bt_mesh_sensor_value`. So what to reference depends on what you want to talk about (sensor values or sensor formats)

Reply:
In line 698 it is correct with the struct bt_mesh_sensor_format, they are different.